vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -577,6 +578,51 @@ func resourceVSphereComputeCluster() *schema.Resource {
 					},
 				},
 			},
+			"vsan_stretched_cluster_enabled": {
```

Comment:
Any chance we can combine semantics of the 2 attributes? Like if there are zero entries in the typeset is it disabled? If there ARE entries in the TypeSet enable it?

This would probably mean the "conf" attribute can't be Computed so Terraform knows when you remove the config we want to explicitly disable this feature.

Is that possible or is the feature more nuanced?

Reply:
The reason we split to two fields is that besides basic enable/disable stretched cluster, there're other workflows like reconfigure, users could move hosts from one fd to another, and witness node could also be replaced, these workflows use different APIs. This part would be implemented in future change.
If we combine these 2 attributes, I'm thinking how to distinguish if it's the first time we enable stretched cluster or reconfigure needed actually. That's why we create a switch for that.

Reply:
have combined 2 attributes in latest commit.